sim/side.ts

```diff
@@ -272,6 +272,7 @@ export class Side {
 		const slotConditionData = this.slotConditions[target][status.id] = {
 			id: status.id,
 			target: this,
+			targetPosition: target,
```

Comment:
when is this used?

Reply:
Never, but I thought it might be nice to have? I'll remove it.

Reply:
You can keep it if you'd like, I just wanted to make sure I wasn't missing something :)

Reply:
Someone can re-add it in the future if for some reason it becomes necessary, but if it does then something is probably wrong with how slot conditions are implemented.